.github/workflows/validate-release-pr.yaml

```diff
@@ -0,0 +1,25 @@
+name: Validate Release PR
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  validate-release:
+    if: startsWith(github.head_ref, 'release/automated-')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Check if release PR is up-to-date
+        run: |
+          COMMITS_BEHIND=$(git rev-list --count HEAD..origin/master)
```

Comment:
no, this is a very weak check, because one can do a merge commit from master and that will make it succed. 
you need to run lerna version on master I think, same as in create-release-pr, and compare with this branch

Reply:
I also imagine that the real security comes from @cognitedata/javascript-sdk-maintainers as reviewers - double checking that the release PR is up to date. I'm not sure we should add too high complexity systems to maintain, atleast immediately.